packages/http-client-csharp/generator/TestProjects/Local/Unbranded-TypeSpec/src/Generated/Internal/ClientPipelineExtensions.cs

```diff
@@ -63,5 +63,10 @@ public static ClientResult<bool> ProcessHeadAsBoolMessage(this ClientPipeline pi
                     return new ErrorResult<bool>(response, new ClientResultException(response));
             }
         }
+
+        private static PipelineResponse ExtractResponseContent(PipelineMessage message)
+        {
+            return message.ExtractResponse();
+        }
```

Comment:
is this change necessary? it looks quite redundant.
Or maybe we could add a `AggressiveInlining` attribute to this method

Reply:
For Azure, we need to have a method other than a single statement: https://github.com/Azure/azure-sdk-for-net/pull/46530/files#diff-fd2884ff6120bf1566d7b7d990188218e9069b472193167d7a53d80ddc2116a8